src/components/ViewDao/DaoRedemptionsContainer.tsx

```diff
@@ -110,22 +114,29 @@ export default (props: { dao: IDAOState, currentAccountAddress: Address } & Rout
     proposals(where: {
       accountsWithUnclaimedRewards_contains: ["${props.currentAccountAddress}"]
       dao: "${props.dao.address}"
-      contributionReward_not: null
+      stage_in: ["Boosted", "QuietEndingPeriod", "Executed"]
```

Comment:
why would we want boosted proposals here?

Reply:
oops, good point.

Just the "executed" proposals, then

Reply:
when we upgrade the the new version of the subgraph, this line will be unecessary : https://github.com/daostack/subgraph/issues/252